libs/content/workspace/Cargo.toml

```diff
@@ -18,7 +18,7 @@ epaint = "0.28.1"
 glam = "0.22.0"
 image = "0.24"
 lb-fonts = { git = "https://github.com/lockbook/lb-fonts" }
-hayro = "0.4.0"
+hayro = "0.3.0"
```

Comment:
@Parth was this intentional?

Reply:
I fixed in a subsequent commit a couple minutes later